ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -46,6 +46,15 @@ aaa_server_groups:
       - server: 192.168.10.157
         vrf: mgt
       - server: 10.10.10.248
+  - name: RADIUS2
```

Comment:
Try changing this to TEST_RADIUS2 so it sorts _after_ TACACS. That will show the chained sort issue I mentioned.

Reply:
Yes, with my logic, it was sorting on name first. I swapped them.

Removed these tests as I will add the changes of ordering in other templates.